Scenario/scenario_shacl.ttl

```diff
@@ -0,0 +1,439 @@
+@prefix scenario: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/scenario/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+scenario:scenarioShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:EnvironmentalShape ;
+            sh:order 2 ;
+            sh:path scenario:Environmental ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:GeneralShape ;
+            sh:order 0 ;
+            sh:path scenario:General ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:Data_SourcesShape ;
+            sh:order 6 ;
+            sh:path scenario:Data_Sources ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:StructuralShape ;
+            sh:order 4 ;
+            sh:path scenario:Structural ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:TrafficShape ;
+            sh:order 3 ;
+            sh:path scenario:Traffic ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:ParticipantsShape ;
+            sh:order 1 ;
+            sh:path scenario:Participants ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:FormatShape ;
+            sh:order 5 ;
+            sh:path scenario:Format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:GeoreferenceShape ;
+            sh:order 7 ;
+            sh:path scenario:Georeference ] ;
+    sh:targetClass scenario:scenario .
+
+scenario:GeoreferenceShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:ProjectLocationShape ;
+            sh:order 0 ;
+            sh:path scenario:ProjectLocation ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node scenario:GeodeticReferenceSystemShape ;
+            sh:order 1 ;
+            sh:path scenario:GeodeticReferenceSystem ] ;
+    sh:targetClass scenario:georeference .
+
+scenario:GeodeticReferenceSystemShape a sh:NodeShape ;
+    sh:property [ skos:example "Lat: 1.234; Lon 45.23" ;
+            sh:description "World coordinates of map origin"@en ;
+            sh:message "Validation of origin failed!"@en ;
+            sh:name "origin"@en ;
+            sh:node scenario:Coordinate2DShape ;
+            sh:minCount 1 ;
+            sh:maxCount 1 ;
+            sh:order 2 ;
+            sh:path scenario:origin ],
+        [ skos:example "EPSG 32632" ;
+            sh:datatype xsd:string ;
+            sh:description "EPSG code of the map"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of coordinateSystem failed!"@en ;
+            sh:name "coordinateSystem"@en ;
```

Comment:
can you provide an `sh:in` to define valid values?

Reply:
Have done it, but more for demonstration purposes. Generally speaking, it might make more sense referring to the offical specification, e.g. https://epsg.org/home.html
Maybe @3Dbastian can help here out?